Added info about playbooks for collection (#75535)
ansible/ansible: docs/docsite/rst/dev_guide/developing_collections_structure.rst

playbooks directory
--------------------

> Since 2.9 you can reference playbooks in this directory, but it requires the full path to the file from the command line.

Reviewer: ```suggestion
In prior releases, you could reference playbooks in this directory using the full path to the playbook file from the command line.
```

> Since ansible-core 2.11 you can use the FQCN, ``namespace.collection.playbook`` (with or without extension), to reference them from the command line or from ``import_playbook``.

Reviewer: ```suggestion
In ansible-core 2.11 and later, you can use the FQCN, ``namespace.collection.playbook`` (with or without extension), to reference the playbooks from the command line or from ``import_playbook``.
```

This will keep the playbook in 'collection context', as if you had added ``collections: [ namespace.collection ]`` to it.

> You can have most of the subdirectories you would expect, like ``files/``, ``vars/`` or  ``templates/`` but no ``roles/`` since those are handled already in the collection.

Reviewer: ```suggestion
You can have most of the subdirectories you would expect, such ``files/``, ``vars/`` or  ``templates/`` but no ``roles/`` since those are handled already in the collection.
```
